docs/changelog/109487.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 109487
+summary: StartTrainedModelDeployment request query params now override body params
```

Comment:
How about we remove the reference to the class name (users won't know what that means). Maybe something like `Star trained model deployment API requests query params now override body params.